build.sh

```diff
@@ -24,7 +24,6 @@ usage() {
   echo "  -f  force a clean build             DEFAULT: NO"
   echo "  -d  include JCSDA ctest data        DEFAULT: NO"
   echo "  -a  build everything in bundle      DEFAULT: NO"
-  echo "  -m  select dycore                   DEFAULT: FV3"
```

Comment:
@TingLei-NOAA , why remove this option and not just change to default to MPAS?

Reply:
I need to re-sort out on this part of this part.  Thanks.